pkg/profiler/profiler.go

```diff
@@ -35,6 +35,45 @@ type StackID struct {
 	TID PID
 }
 
+
+// TODO[btv]
```

Comment:
I would think Go should be the source of truth and go generate should be used to generate a header.  But others have probably trodden this path, I wonder what cilium/ebpf does.